### ISSUE TITLE
Nothing in activity log

### ISSUE THREAD

**Issue by jonwcrew** [nick-prater/kronekeeper#50]
I tried adding  a block and some jumpers to a frame .. I don't see anything in the new activity log view

**Comment by nick-prater** [comment]
I'm seeing activity in the CTA2 Frame that looks like it might be you... Are you not seeing this? Entirely possible I've broken something.

![activity_log](https://cloud.githubusercontent.com/assets/1817335/21825712/ba43a3c2-d77c-11e6-98e7-fd0dead5d8d8.png)

**Comment by jonwcrew** [comment]
Yes that’s me .. no I don’t see the show drop down or the entries below.

Mac safari and chrome tested

Screen shot attached.

**Comment by nick-prater** [comment]
I've been unable to replicate the problem on Windows or Linux. Let me take a look on a Mac at LQ tomorrow.

**Comment by jonwcrew** [comment]
Nick .. following conversation today. I think these things are needed to make the activity log minimally useful as a jumping list that you might hand to a wireman for example.

Have ranked in phases of usefulness

v1
Filter activity list to show only jumper insertions and deletions
Filter by the user (so you can extract only your work)
Sort to present deletions first
Export to text file .. or indeed any useful file type .. or get the list out be any means.

v2
Auto sort list to group into a useful order .. i.e. group jumpers by block sorted circuit 1 to 10
Print button in browser

v3
Live jumping interface that you use on a tablet or phone.

**Comment by nick-prater** [comment]
Enabled on demo system:
  * Filter activity list to show only jumper insertions and deletions
  * Filter by user
  * Export to xlsx file

Please test and let me have your feedback.

I think the text shown in the Activity log needs to be revised so that it is a concise instruction. Rather than "standard jumper added B76.0->B77.0 [blue/red]" perhaps "Add jumper B76.0->B77.0 [blue/red]"

Sorting to present deletions first not currently implemented. This needs some thought as it involves more than just sorting if a long sequence is being presented. For example what if the sequence is jumper added, removed, different jumper added. In this case, the first jumper add/remove needs to be filtered out completely.

**Comment by nick-prater** [comment]
And I'm now doing some work so that the activity log can associate jumper additions and deletions
